lib/autoproj/cli/envsh.rb

```diff
@@ -4,7 +4,7 @@ module CLI
         class Envsh < InspectionTool
             def validate_options(_unused, options = Hash.new)
                 _, options = super(_unused, options)
-                options
+                [options]
```

Comment:
are there already tests for cli/test_envsh?

Reply:
Added one

Reply:
added one